src/prepack-options.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import type { ErrorHandler } from "./errors.js";
+import type { SerializerOptions, RealmOptions } from "./options";
+
+export type Compatibility = "browser" | "jsc-600-1-4-17" | "node-source-maps" | "node-cli";
+export const CompatibilityValues = ["browser", "jsc-600-1-4-17", "node-source-maps", "node-cli"];
+
+export type Options = {|
```

Comment:
You could rename this type to `PrepackOptions`, or `CombinedPrepackOptions`, to make it a bit more clear what this type represents.